readthedocs.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+formats:
+  - htmlzip
+  - pdf
+python:
+  version: 3.7
+  install:
+    - requirements: docs/env-docs.yml
+  system_packages: true
+
+sphinx:
+  builder: html
+  configuration: docs/conf.py
+  fail_on_warning: false
```

Comment:
This file is redundant as there's `.readthedocs.yml` in this repo. Is there a specific reason to add this file?

Reply:
🤦‍♀️ I think I missed that! Fixed now